db/migrate/20220809135529_add_users_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
- [ ] According to the ERD diagram, the reference is for user table with an alias of author. Please ensure that this is implemented.

![](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png?raw=true)